src/Console/ShellDispatcher.php

```diff
@@ -279,7 +279,6 @@ public function addShortPluginAliases(): array
 
             $other = static::alias($shell);
             if ($other) {
-                $other = $aliases[$shell];
```

Comment:
This was the bug - a no-op as $other is being assigned $plugin value and then compared afterwards.